satpy/modifiers/atmosphere.py

```diff
@@ -60,6 +94,10 @@ def __call__(self, projectables, optional_datasets=None, **info):
 
         atmosphere = self.attrs.get('atmosphere', 'us-standard')
         aerosol_type = self.attrs.get('aerosol_type', 'marine_clean_aerosol')
+        reduce_lim_low = abs(self.attrs.get('reduce_lim_low', 70))
+        reduce_lim_high = abs(self.attrs.get('reduce_lim_high', 95))
```

Comment:
I'm a bit concerned about this hard limit. Why was 95 degrees chosen? In some conditions you can have meaningful radiances beyond 95 degrees so if there's no important reason for this choice then I'd like to see the hard upper limit bumped to something like 105 degrees.

Reply:
Ok we'll set lim_high 105° by default